lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/checkbox.scss

```diff
@@ -0,0 +1,85 @@
+.checkbox {
```

Comment:
Jake and I couldn't decide if this should be split into its own `.scss` file or if it should be included in `form.scss` with the class name `.form__checkbox`. The same goes for `radio.scss`